frontend/src/components/Modals/DetailsButtonModal.tsx

```diff
@@ -47,6 +49,9 @@ export const DetailsButtonModal = ({
     <Modal show={show} onHide={handleClose} fullscreen={true}>
       <Modal.Header closeButton>
         <Modal.Title>
+          <small className="text-muted ps-2">
+            JSON Schema test cases that need attention : &nbsp;
```

Comment:
What's with the nonbreaking spaces? A few of your PRs have had them.

Reply:
I use &nbsp to have a space between text and value, when there is text preceding a value. I don't want the value to stick to the colon, like :Json-Java-Schema. Instead, I would like to have it show like : Json-Java-Schema.

Reply:
A nonbreaking space is very weird to use for that -- I'm no frontend developer so I don't know whether there are any norms there -- but you sound like you just want "a space". The nonbreaking behavior has nothing to do with anything, and worse is probably wrong -- if need be we probably *would* want to break at that character.

If having a space isn't working that (to me, again not a frontend expert) sounds like a CSS bug, since one (i.e. exactly one) should be being preserved -- presumably using `<label>` or some other proper tag to mark the label would have fixed this as well.

Reply:
I certainly can use labels with padding properties. The project has not used label tag on any screen so far and I tried to keep my fixes in synch with the existing coding style. I will in the future with your permission do what is right for my fixes even if it looks slightly out of place. Sorry about the back and forth.

Reply:
I'll include labels and check in code.